.github/workflows/gpu_pcc.yml

```diff
@@ -86,6 +86,11 @@ jobs:
             echo "HOME=/home/ubuntu";
           } >> "${GITHUB_ENV}"
 
+      - name: Install libclang
+        if: ${{ !cancelled() }}
+        run: |
+          sudo apt-get update && sudo apt-get install -y libclang-dev
```

Comment:
just a little more

Reply:
This one is the only one done on AWS, @soonum could be worth having a new AMI with libclang-dev installed wdyt?

Reply:
Maybe it's already installed on the AMI, I'm checking just in case

Reply:
All good, we don't need a new AMI! :tada:

Reply:
Oh, the binding was not being generated so that's why I wasn't getting an error. Now I get one :disappointed:

Reply:
I've created a new AMI, Concrete-cuda / tfhe-rs v10 cc @soonum